tests/src/test/scala/system/health/BasicHealthTest.scala

```diff
@@ -67,33 +72,39 @@ class BasicHealthTest
 
     it should "fire a trigger when a message is posted to message hub" in withAssetCleaner(wskprops) {
         val currentTime = s"${System.currentTimeMillis}"
+        val triggerName = "/_/BasicHealthTestTrigger"
 
         (wp, assetHelper) =>
-            val triggerName = s"/_/dummyMessageHubTrigger-$currentTime"
-            println(s"Creating trigger ${triggerName}")
-
-            val feedCreationResult = assetHelper.withCleaner(wsk.trigger, triggerName) {
-                (trigger, _) =>
-                    trigger.create(triggerName, feed = Some(s"$messagingPackage/$messageHubFeed"), parameters = Map(
-                        "user" -> kafkaUtils.getAsJson("user"),
-                        "password" -> kafkaUtils.getAsJson("password"),
-                        "api_key" -> kafkaUtils.getAsJson("api_key"),
-                        "kafka_admin_url" -> kafkaUtils.getAsJson("kafka_admin_url"),
-                        "kafka_brokers_sasl" -> kafkaUtils.getAsJson("brokers"),
-                        "topic" -> topic.toJson))
-            }
+            val result = wsk.trigger.get(triggerName, expectedExitCode = DONTCARE_EXIT)
 
-            withActivation(wsk.activation, feedCreationResult, initialWait = 5 seconds, totalWait = 60 seconds) {
-                activation =>
-                    // should be successful
-                    activation.response.success shouldBe true
-            }
+            if(result.exitCode == NOT_FOUND) {
+                // trigger does not yet exist, create it
+                println(s"Creating trigger ${triggerName}")
 
-            // It takes a moment for the consumer to fully initialize.
-            println("Giving the consumer a moment to get ready")
-            Thread.sleep(consumerInitTime)
+                val feedCreationResult = wsk.trigger.create(triggerName, feed = Some(s"$messagingPackage/$messageHubFeed"), parameters = Map(
+                    "user" -> kafkaUtils.getAsJson("user"),
+                    "password" -> kafkaUtils.getAsJson("password"),
+                    "api_key" -> kafkaUtils.getAsJson("api_key"),
+                    "kafka_admin_url" -> kafkaUtils.getAsJson("kafka_admin_url"),
+                    "kafka_brokers_sasl" -> kafkaUtils.getAsJson("brokers"),
+                    "topic" -> topic.toJson))
+
+                withActivation(wsk.activation, feedCreationResult, initialWait = 5 seconds, totalWait = 60 seconds) {
+                    activation =>
+                        // should be successful
+                        activation.response.success shouldBe true
+                }
+
+                // It takes a moment for the consumer to fully initialize.
+                println("Giving the consumer a moment to get ready")
+                Thread.sleep(consumerInitTime)
+            } else {
+                result.exitCode shouldBe(0)
```

Comment:
Should we make this `result.exitCode shouldBe SUCCESS_EXIT` (you're using `NOT_FOUND` above already)